system-test/system-test-script.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#Stopping the script if any command fails
+set -euo pipefail
+export myvar="system-test/logfile"
+cd ..
```

Comment:
let's make the assumption that this script will be run from the root dir instead of moving around. You can add that to the docs that it's meant to be run like `system-tests/script.sh` or whatever, that way you won't need to `cd ..`

Reply:
fixed.